README.md

```diff
@@ -6,96 +6,94 @@
 [![GitHub Workflow Status](https://img.shields.io/github/workflow/status/CanalTP/transit_model/Continuous%20Integration?logo=github&style=flat-square)](https://github.com/CanalTP/transit_model/actions?query=workflow%3A%22Continuous+Integration%22)
 [![License: AGPL v3.0](https://img.shields.io/github/license/CanalTP/transit_model?color=9873b9&style=flat-square)](./LICENSE)
 
-`transit_model` is a Rust crate managing transit data by implementing the NTFS
-model (used  in [navitia](https://github.com/CanalTP/ntfs-specification/blob/master/ntfs_fr.md)). See the
-section [NTFS: Level of Support](#ntfs-level-of-support) for more details about the
-level of support of the NTFS standard.
-
-This repository also provides :
-- (incomplete) [GTFS](http://gtfs.org/) to [NTFS](https://github.com/CanalTP/ntfs-specification/blob/master/ntfs_fr.md) and (soon) NTFS to GTFS conversion.
-- (incomplete) Generation of transfers.
-.
-- Merge [NTFS] (https://github.com/CanalTP/ntfs-specification/blob/master/ntfs_fr.md).
-
-## Compile
-
-```bash
-cargo build --release
+**`transit_model`** is a Rust crate to manage, convert and enrich transit
+data.<br>
+This is done by implementing the [NTFS] model (used in [navitia]).
+
+This repository regroups crates that offer enabler-libraries and binaries to
+convert and enrich transit data.
+
+Please check documentation attached to each crate:
+* [**gtfs2netexfr**](gtfs2netexfr/README.md) converts [GTFS] data format into
+  [NeTEx]-France data format.
+* [**gtfs2ntfs**](gtfs2ntfs/README.md) converts [GTFS] data format into [NTFS]
+  data format.
+* [**ntfs2gtfs**](ntfs2gtfs/README.md) converts [NTFS] data format into [GTFS]
+  data format.
+* [**ntfs2netexfr**](ntfs2netexfr/README.md) converts [NTFS] data format into
+  [NeTEx]-France data format.
+* [**ntfs2ntfs**](ntfs2ntfs/README.md) checks and cleans a [NTFS] dataset.
+* [**restrict-validity-period**](restrict-validity-period/README.md) restricts
+  the validity period of a [NTFS] dataset and purges out-of-date data.
+
+## [PROJ] dependency
+
+Based on [PROJ], the [`proj` crate] allows the transformation of
+localization coordinates.
+
+Some `transit_model`'s crates (see each documentation) use [PROJ].<br>
+So it must be installed on the system to compile and use those crates.
+
+### [PROJ] for binaries
```

Comment:
We have to specify somewhere that the instructions are for a Debian base (we already had the remark in the past).